examples/utils/migration-tools/src/simpleLoader/sessionStorageSimpleLoader.ts

```diff
@@ -53,9 +50,11 @@ export class MigratableSessionStorageModelLoader<ModelType>
 		return true;
 	}
 
-	public async createDetached(version: string): Promise<IDetachedMigratableModel<ModelType>> {
+	public async createDetached(
+		version: string,
+	): Promise<{ container: IContainer; attach: () => Promise<string> }> {
 		const documentId = uuid();
-		const modelLoader = new MigratableModelLoader<ModelType>({
+		const modelLoader = new SimpleLoader({
```

Comment:
We should probably update all these `modelLoader` names
```suggestion
		const loader = new SimpleLoader({
```